src/main/resources/application.properties

```diff
@@ -10,4 +10,5 @@ spring.jpa.hibernate.ddl-auto = update
 
 server.port = 8081
 
-services.openLibraryBooksBaseUrl = https://openlibrary.org/api
\ No newline at end of file
+services.openLibraryBooksBaseUrl = http://localhost:8080
+services.openLibraryBooksBaseUrl2 = https://openlibrary.org/api
```

Comment:
I'm not sure why do you need these two open library urls but maybe this [link](https://www.baeldung.com/spring-profiles#profiles-in-spring-boot) could be helpful. You can have an application.properties for each profile